packages/search/src/features/registration/deduplicate/test-util.ts

```diff
@@ -47,41 +44,59 @@ type ComparisonObject<T> = {
 type Values<T> = T[keyof T]
 
 export async function compareForBirthDuplication(
-  registrationComparison: ComparisonObject<IBirthCompositionBody>,
+  registrationComparison: ComparisonObject<
+    Omit<BirthDocument, 'compositionId'>
+  >,
   client: elasticsearch.Client
 ) {
   const existingComposition = Object.fromEntries(
-    Object.entries<Values<IBirthCompositionBody>[]>(registrationComparison).map(
+    Object.entries<Values<BirthDocument>[]>(registrationComparison).map(
       ([key, values]) => [key, values[0]]
     )
   )
   const newComposition = Object.fromEntries(
-    Object.entries<Values<IBirthCompositionBody>[]>(registrationComparison).map(
+    Object.entries<Values<BirthDocument>[]>(registrationComparison).map(
       ([key, values]) => [key, values[1]]
     )
   )
 
-  await indexComposition('123-123-123-123', existingComposition, client)
-  const results = await searchForBirthDuplicates(newComposition, client)
+  await indexComposition(
+    '123-123-123-123',
+    { compositionId: '123-123-123-123', ...existingComposition },
+    client
+  )
+  const results = await searchForBirthDuplicates(
+    { compositionId: '123-123-123-123', ...newComposition },
```

Comment:
does it make sense for the newComposition to have an id already as it's not stored yet?

Reply:
hmm. can we save records to ElasticSearch that don't have an identifier? We wouldn't be able to link them to the Mongo records